nomad/local/run_integration_tests.sh

```diff
@@ -17,10 +17,19 @@ echo "--- Dispatching integration tests"
 job_id=$(nomad_dispatch integration-tests)
 echo "${job_id}"
 
-await_nomad_dispatch_finish "${job_id}" $((5 * 60))
+dispatch_timed_out=0
+await_nomad_dispatch_finish "${job_id}" $((5 * 60)) && dispatch_timed_out=1
```

Comment:
Shouldn't this be || not &&? I also suspect that await_nomad_dispatch_finish should exit 1 if it times out, not return

Reply:
re: the &&/||
yes, good call, thanks.

> I also suspect that await_nomad_dispatch_finish should exit 1 if it times out, not return
if it exits, then this whole script exits, is my understanding.
However, I still want to perform the `if dispatch_timed_out` behavior if that occurs.
As such, sticking with `return`